src/main/java/com/devotedmc/ExilePearl/listener/PlayerListener.java

```diff
@@ -809,7 +847,7 @@ public void onPlayerInteract(PlayerInteractEvent e) {
 		if (e.getAction() == Action.RIGHT_CLICK_BLOCK) {
 			Material m = e.getClickedBlock().getType();
 			if (e.getClickedBlock().getState() instanceof BlockInventoryHolder ||
-				m == Material.CRAFTING_TABLE || m == Material.ANVIL || m == Material.ENCHANTING_TABLE || m == Material.BEACON) {
+					m == Material.CRAFTING_TABLE || m == Material.ANVIL || m == Material.ENCHANTING_TABLE || m == Material.BEACON) {
```

Comment:
Weird formatting, did we ever get you setup with the checkstyle?

Reply:
No